From: Simon McVittie <smcv@debian.org>
Date: Mon, 22 Jun 2020 15:42:29 +0000
Subject: tests: Don't run at build-time

These tests require that our locale files have been installed properly,
so they need to be deferred until libgnome-desktop has been installed.
Build-time tests can't rely on that, but installed-tests can.

Bug: https://gitlab.gnome.org/GNOME/gnome-desktop/-/issues/159
Signed-off-by: Simon McVittie <smcv@debian.org>
Applied-upstream: 3.36.4, commit:94a3996373f53c6beb3fdd5c2b30160cb8e94078
Applied-upstream: 3.37.3, commit:985e49662d527daf5417bfbb8be475bc8d64d64a
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=963384
---
 tests/meson.build | 7 +++----
 1 file changed, 3 insertions(+), 4 deletions(-)

diff --git a/tests/meson.build b/tests/meson.build
index df5634b..6401618 100644
--- a/tests/meson.build
+++ b/tests/meson.build
@@ -37,10 +37,9 @@ foreach program: test_programs
     )
   endif
 
-  test(program,
-    exe,
-    env: test_env
-  )
+  # Don't run these tests at build-time with test(): currently both tests
+  # rely on our locales having been installed, and at build-time that
+  # hasn't happened yet.
 endforeach
 
 if get_option('installed_tests')
